specification/marketplacecatalog/resource-manager/Solutions/preview/2023-03-01-preview/solutions.json

```diff
@@ -0,0 +1,482 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Marketplace Solution Builder",
+    "version": "2023-03-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Marketplace/mysolutions": {
+      "get": {
+        "tags": [
+          "UserSolutions"
+        ],
+        "description": "Returns all user solutions",
+        "operationId": "GetUserSolutions",
+        "x-ms-examples": {
+          "GetUserSolutions": {
+            "$ref": "./examples/GetUserSolutions.json"
+          }
+        },
+        "parameters": [
+          {
+            "type": "string",
+            "name": "api-version",
+            "in": "query",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/UserSolutions"
+            }
+          },
+          "default": {
+            "x-nullable": false,
+            "description": "*** Error Responses: ***\n\n * 400 - In case request malformed.\n\n * 500 - In case internal error.",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "post": {
```

Comment:
PUT needs to be used to create resources. ARM RPC standards doesnt allow POST to create resources